src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -440,6 +445,14 @@ export const getFormSchema = (
             "ui:hide": true,
           }),
         },
+        requiredApprovalCount: {
+          "ui:data-cy": "required-approval-count-input",
+          ...((formData?.commitQueue?.enabled === false ||
+            !!githubProjectConflicts?.commitQueueIdentifiers?.length) && {
+            "ui:hide": true,
+          }),
+          ...placeholderIf(repoData?.commitQueue?.requiredApprovalCount),
+        },
```

Comment:
It seems like having this feature disabled will hide the other fields, but not this one. Should the field be hidden when the feature is disabled?

<img width="549" alt="Screen Shot 2023-01-20 at 3 11 44 PM" src="https://user-images.githubusercontent.com/47064971/213797224-92d062fe-d6d9-4282-b7c1-5d02ea4464f3.png">

Reply:
yes, fixed